pyfakefs/tests/fake_pathlib_test.py

```diff
@@ -21,24 +21,55 @@
 python docs.
 """
 
+import errno
 import os
-import pathlib
 import stat
 import sys
 import unittest
 
 from pyfakefs import fake_pathlib, fake_filesystem
-from pyfakefs.tests.test_utils import RealFsTestCase
+from pyfakefs.extra_packages import pathlib, pathlib2
+from pyfakefs.helpers import text_type
+from pyfakefs.tests.test_utils import RealFsTestCase, TestCase
 
 is_windows = sys.platform == 'win32'
 
 
+def needs_pathlib_35():
+    if sys.version_info < (3, 5) and not pathlib2:
+        raise unittest.SkipTest('New in version 3.5')
```

Comment:
Nice - though I would rename these to be consistent with the other skip functions (e.g. either `skip_if_xxx` or `check_if_xxx`).